front50-web/src/main/groovy/com/netflix/spinnaker/front50/config/Front50WebConfig.groovy

```diff
@@ -45,6 +48,11 @@ public class Front50WebConfig extends WebMvcConfigurerAdapter {
   }
 
   @Bean
+  Filter authenticatedRequestFilter() {
+    new AuthenticatedRequestFilter(true)
```

Comment:
This is the filter that actually extracts the auth headers and sticks them in the MDC context.